numpy/random/tests/test_random.py

```diff
@@ -255,6 +255,18 @@ def test_random_integers_max_int(self):
         desired = np.iinfo('l').max
         np.testing.assert_equal(actual, desired)
 
+    def test_random_integers_deprecated(self):
```

Comment:
@charris : "quick and dirty" = move test into ```test_deprecations.py``` or that ```with warnings.catch_warnings()``` block?  Both seem to fit the criterion.